clientd3d/d3dparticle.h

```diff
@@ -8,16 +8,7 @@
 #ifndef __D3DPARTICLE_H__
 #define __D3DPARTICLE_H__
 
-#define MAX_PARTICLES	128
-#define SANDSTORM_R		226
-#define SANDSTORM_G		153
-#define SANDSTORM_B		6
-#define SANDSTORM_A		255
-
-#define RAIN_R			128
-#define RAIN_G			128
-#define RAIN_B			128
-#define RAIN_A			128
+#define MAX_PARTICLES	512 // was 128
```

Comment:
Any reason to preserve the historical value of constants?

Since this makes particle systems 4x more expensive, you'll need to do some performance testing.

Also make this a const int rather than a macro.